src/compiler/file_uploader/hash_status_actor.ts

```diff
@@ -0,0 +1,22 @@
+import { Actor, ActorSubclass } from '@dfinity/agent';
+import { createAuthenticatedAgent } from '../../../dfx';
+
+export async function createActor(canisterId: string): Promise<ActorSubclass> {
```

Comment:
Doesn't this return type remove useful information?

Reply:
done